README.rst

```diff
@@ -74,14 +74,10 @@ Pushing a new release
 
    You may want to check the HISTORY.rst file renders correctly before
    committing, as the twine upload will fail later if it
-   doesn't. Install one of these::
+   doesn't::
 
-    # dnf install python3-readme-renderer
-    $ pip install readme-renderer
-
-   Then run the following command::
-
-    $ python -m readme_renderer HISTORY.rst -o /tmp/HISTORY.html
+    $ python setup.py sdist bdist_wheel
+    $ twine check dist/*
```

Comment:
I think we want this to match what was added to tox.

Reply:
Maybe it makes sense to use the tox job directly, even?

Reply:
Yes using the tox command would get rid of the need for the sdist command too.